templates/webapi/main/group_overview.html.ep

```diff
@@ -1,5 +1,5 @@
 % layout 'bootstrap';
-% title '';
+% title ((defined $group->{parent_name} ? $group->{parent_name}  . " / " : "") . $group->{name});
```

Comment:
I wonder if we use this functionality already elsewhere and shouldn't duplicate but haven't found it yet. Maybe also there is a nicer way to write this without needing the logic?